packages/ast-spec/src/declaration/ExportNamedDeclaration/spec.ts

```diff
@@ -64,9 +71,14 @@ export interface ExportNamedDeclarationWithoutSourceWithMultiple
   attributes: ImportAttribute[];
   declaration: null;
   source: null;
-  specifiers: ExportSpecifier[];
```

Comment:
Removals here are not breaking because these properties are already declared on the base